portal_sale_order_type/README.rst

```diff
@@ -0,0 +1,75 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Portal Sale Order Type
+======================
+This module add a rule to portal user and set sale order type invisible
+ with portal user.
```

Comment:
cambiar por "
Integration betweeen Portal and Sale Order Type module:
#. New access record to sale type model for portal users
#. Make invisible sale order type for portal users